src/Httpful/Response/Headers.php

```diff
@@ -36,7 +46,7 @@ public static function fromString($string)
      */
     public function offsetExists($offset)
     {
-        return isset($this->headers[strtolower($offset)]);
```

Comment:
I believe this would be a breaking change. It's one I'm for but I believe we'll need to wait to do this until the next major release.